frontend/src/views/404.vue

```diff
@@ -1,26 +1,25 @@
 <template>
-  <div class="PageNotFound app flex-row align-items-center">
+  <div class="app flex-row align-items-center">
     <div class="container">
```

Comment:
Not sure but it should be a `<b-container>` tag https://bootstrap-vue.org/docs/components/layout#layout-and-grid-system